src/google/protobuf/compiler/objectivec/objectivec_helpers.cc

```diff
@@ -531,6 +539,11 @@ std::string FileClassPrefix(const FileDescriptor* file) {
     return file->options().objc_class_prefix();
   }
 
+  // If prefix is passed through objc_opt then accept that first.
```

Comment:
I'd probably drop the `first` on this sentence, since it isn't _first_, the file option is _first_, this is just accepted next before the prefix mode.

Reply:
sg, will fix!